CI broken

**Describe the bug**

CI tests currently fail presumably caused by dependency updates:

```
=========================== short test summary info ============================
FAILED tests/config/test_validate.py::ConfigValidateTest::test_validate_variable_fail - AssertionError: Regex pattern did not match.
 Regex: "Invalid configuration: 10 is not of type 'string' for variables.chl.dims.0"
 Input: "Invalid configuration: 30 is not of type 'string' for variables.chl.dims.2"
FAILED tests/test_xrencoding.py::XArrayEncodingTest::test_overlapping_chunks_in_data_and_encoding_raises - ValueError: Specified zarr chunks encoding['chunks']=(20,) for variable named 'v' would overlap multiple dask chunks ((10, 10, 10, 10, 10, 10, 10, 10, 10, 10),). Writing this array in parallel with dask could lead to corrupted data. Consider either rechunking using `chunk()`, deleting or modifying `encoding['chunks']`, or specify `safe_chunks=False`.
================= 2 failed, 215 passed, 122 warnings in 19.36s =================
```

**To Reproduce**

See https://github.com/bcdev/zappend/actions/runs/9269795967/job/25501354560#step:6:456

**Python Environment**

CI env, all supported Python versions

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,8 @@
 
 * Fixed link to _slice sources_ in documentation main page.
 
+* Fixed broken CI. (#97)
+
 
 ## Version 0.7.0 (from 2024-03-19)
 

--- a/tests/config/test_validate.py
+++ b/tests/config/test_validate.py
@@ -43,7 +43,7 @@ class ConfigValidateTest(unittest.TestCase):
         with pytest.raises(
             ValueError,
             match="Invalid configuration:"
-            " 10 is not of type 'string'"
-            " for variables.chl.dims.0",
+            " [1-3]0 is not of type 'string'"
+            " for variables.chl.dims.[0-2]",
         ):
             validate_config(config)

--- a/tests/slice/test_cm.py
+++ b/tests/slice/test_cm.py
@@ -7,6 +7,11 @@ import shutil
 import unittest
 import warnings
 
+try:
+    # noinspection PyUnresolvedReferences
+    import h5netcdf
+except ModuleNotFoundError:
+    h5netcdf = None
 import pytest
 import xarray as xr
 
@@ -20,9 +25,6 @@ from zappend.slice.sources.persistent import PersistentSliceSource
 from zappend.slice.sources.temporary import TemporarySliceSource
 from tests.helpers import clear_memory_fs
 from tests.helpers import make_test_dataset
-
-
-# noinspection PyUnusedLocal
 
 
 # noinspection PyShadowingBuiltins,PyRedeclaration,PyMethodMayBeStatic
@@ -95,6 +97,7 @@ class OpenSliceDatasetTest(unittest.TestCase):
             # root cause. But it may be related to just the memory FS.
             warnings.warn(f"received known exception from to_netcdf(): {e}")
 
+    @unittest.skipIf(h5netcdf is None, reason="h5netcdf not installed")
     def test_slice_item_is_uri_of_local_fs_nc(self):
         engine = "h5netcdf"
         format = "NETCDF4"

--- a/tests/test_xrencoding.py
+++ b/tests/test_xrencoding.py
@@ -130,7 +130,7 @@ class XArrayEncodingTest(unittest.TestCase):
         self.ds["v"] = self.ds.v.chunk(x=10)
         self.ds.v.encoding.update(chunks=(20,))
         with pytest.raises(
-            NotImplementedError,
+            (NotImplementedError, ValueError),
             match="Specified zarr chunks"
             " encoding\\['chunks'\\]=\\(20,\\)"
             " for variable named 'v'",

